tests/testinterop.nim

```diff
@@ -288,6 +288,49 @@ suite "Interop":
     await daemonNode.close()
     await sleepAsync(1.seconds)
 
+  asyncTest "daemon -> websocket connection":
```

Comment:
Shouldn't we also have a complement to this, where we dial the go daemon on the WS?